servers/zms/src/main/resources/messages/domain-member-review.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <style></style>
+</head>
+<body style="margin: 0; background: rgba(231, 232, 231, 1.0);">
+<div class="athenz-wrapper">
+    <div class="mbrapproval">
+        <div class="logo">
+            <img src="cid:logo" class="athenzlogowhite" alt="Athenz logo"/>
+        </div>
+        <div class="hdr">Domain Membership Review Details</div>
+        <div class="bt">You have one or more principals in your Athenz domain <b>{0}</b> whose review date will pass soon:</div>
+        <hr>
+        <table id="t02">
+            <thead>
+            <tr>
+                <th class="ch">MEMBER</th>
+                <th class="ch">ROLE</th>
+                <th class="ch">EXPIRATION</th>
```

Comment:
we should remove EXPIRATION from this reminder. The user will get those separately so let's not combine and possibly confuse the users.